src/SonarQube.Client/Services/SonarQubeService.cs

```diff
@@ -98,6 +98,8 @@ public bool HasOrganizationsFeature
 
         public void Disconnect()
         {
+            EnsureIsConnected();
```

Comment:
We shouldn't check for that. It's like disposing twice - that should be a safe operation.

Reply:
I agree.